third_party/jitterentropy/CMakeLists.txt

```diff
@@ -34,7 +34,13 @@ else()
     endif()
 endif()
 
+if(BORINGSSL_PREFIX)
+  set(JITTER_ENETROPY_PREFIX_INCLUDE "--include=${PROJECT_BINARY_DIR}/symbol_prefix_include/openssl/boringssl_prefix_symbols.h")
```

Comment:
I believe this should be `boringssl_prefix_symbols_asm.h`. You might also need to ensure that `${PROJECT_BINARY_DIR}/symbol_prefix_include/` is one of the include search paths.

Reply:
Nevermind -- `boringssl_prefix_symbols_asm.h` is only needed for processing assembly files.